### DISCUSSION TITLE
scATAC cluster labelling

### DISCUSSION THREAD

**Discussion by komalbollepogu** [GreenleafLab/ArchR#1588] [Questions / Documentation] [answered]
Hello,

Thank you for the wonderful tool. I am analyzing my scATAC data and I already know the clusters. I am wondering how I can annotate/name the clusters without using scRNA data?  Can you please let me know? Thanks so much!

**Comment by rcorces** [comment, marked as answer]
add a column to `cellColData` containing your annotations

**Reply by komalbollepogu** [reply]
Thank you so much!
